application/common/models/EmailQueue.php

```diff
@@ -149,42 +183,47 @@ private function getMailer()
 
     /**
      * Creates an EventLog entry if $this->event_log_* properties are set
+     * @return boolean
+     * @throws \Exception
      */
     private function createEventLogEntry()
     {
         if ($this->event_log_topic !== null && $this->event_log_details !== null && $this->event_log_user_id !== null) {
             EventLog::log($this->event_log_topic, $this->event_log_details, $this->event_log_user_id);
```

Comment:
You might want to add a `return true;` after this line, now that there's a `return false;` when no `EventLog` entry was written.